ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -32,6 +38,8 @@
 HEALTHCHECK_PKG = ["*ipa-healthcheck"]
 TOMCAT_CFG = "/var/lib/pki/pki-tomcat/conf/ca/CS.cfg"
```

Comment:
nit: you can instead uses paths.CA_CS_CFG_PATH

Reply:
made the changes to use paths.CA_CS_CFG_PATH